conf/config.yaml

```diff
@@ -56,6 +56,11 @@ apisix:
   # port_admin: 9180              # use a separate port
   # https_admin: true             # enable HTTPS when use a separate port for Admin API.
                                   # Admin API will use conf/apisix_admin_api.crt and conf/apisix_admin_api.key as certificate.
+  mtls:
```

Comment:
This is not a good name.
 mTLS can be between apisix and upstream, or between dashboard and admin api, which have different meanings.

Reply:
And we need to add support mTLS between apisix and upstream later.